### DISCUSSION TITLE
Circular Map

### DISCUSSION THREAD

**Discussion by jhein05** [zwave-js/zwave-js-ui#1471] [Ideas]
Especially with lots of nodes, I feel like a circular map would be the best way of visualizing the network, because the linear one becomes really wide. I attached an example (from Nagios).

![image](https://user-images.githubusercontent.com/5014124/126669179-3a6d75dc-be54-4352-9f6c-9490f361bd36.png)

**Comment by robertsLando** [comment]
ATM seems mesh graph is still not so accurate, so I see no reason to invest much time refactoring it. Anyway if you would like to submit a PR with that I would be happy to review it, I like the idea

**Reply by jhein05** [reply]
That makes sense. Unfortunately, not being a developer myself, the idea is all I am able to submit. 

Do you have a way of collecting/storing ideas for the future, like an issue with a certain tag or should we just leave it here?

**Reply by robertsLando** [reply]
You can leave it here, if someone else comes here we can discuss about this
